os_migrate/roles/conversion_host/tasks/main.yml

```diff
@@ -117,3 +117,40 @@
     ca_cert: "{{ os_migrate_conversion_ca_cert|default(omit) }}"
     client_cert: "{{ os_migrate_conversion_client_cert|default(omit) }}"
     client_key: "{{ os_migrate_conversion_client_key|default(omit) }}"
+
+- name: create the conversion host floating IP with a fixed IP
+  openstack.cloud.floating_ip:
+    server: "{{ os_migrate_conversion_host_name }}"
+    network: "{{ os_migrate_conversion_external_network_name }}"
```

Comment:
The network parameter should be removed from the case with floating_ip_address, they are mutually exclusive. I looked up the code and if `floating_ip_address` is specified, the module itself will drop `network`, but i think if we do another edit here, it would be nicer to drop the parameter here and not rely on the module doing it. (It could just as well behave the other way - ignore floating_ip_address if network is provided, we are just lucky here.)

https://github.com/openstack/ansible-collections-openstack/blob/ae4e7f3c0607ccf2725b25a6d7d321eb3c32e6be/plugins/modules/floating_ip.py#L269

Reply:
done